index.php

```diff
@@ -70,10 +70,10 @@
             <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                 <ul class="nav navbar-nav navbar-right" style="font-family: 'Montserrat', sans-serif;">
                     <li>
-                        <a class="page-scroll" href="aboutus/index.php">About</a>
+                        <a class="page-scroll" href="aboutus/index.php" style="color: rgb(224,255,255)">About</a>
```

Comment:
* Don't write inline styles. Merge them with css. 
* Don't use `rgb`. Use `hex` based colors. 
* This makes changes at all screen sizes (Even at desktop too). I need these changes only at mobile.